src/app/public/modules/lookup/lookup.component.html

```diff
@@ -56,7 +57,6 @@
             (blur)="onAutocompleteBlur()"
             (keydown)="inputKeydown($event, $event.target.value)"
             (keyup)="inputKeyup($event)"
-            #lookupInput
```

Comment:
Was this template variable just not needed?

Reply:
I believe so. Erika made this change but I can't find any reference to it in either the old or new code so I think we are good here.